packages/controllers/jest.config.js

```diff
@@ -8,10 +8,10 @@ module.exports = {
   coveragePathIgnorePatterns: ['/node_modules/', '/mocks/', '/test/'],
   coverageThreshold: {
     global: {
-      branches: 67.64,
-      functions: 79.88,
-      lines: 81.93,
-      statements: 82,
+      branches: 68.08,
+      functions: 79.63,
+      lines: 81.47,
+      statements: 81.54,
```

Comment:
Why does this lower the coverage?

Reply:
Because I removed tested code I assume? 🤔